### ISSUE TITLE
issues when finetuning the pretrained models for GQA

### ISSUE THREAD

**Issue by runzeer** [airsplay/lxmert#45]
There exists the random seed generator in your codes. And I  found it initially set as 9595. And I also found the parameters you loaded the pretrained models are unchanged if I change the random seed ,for example 2. But the final results on val datasets are worse, less than 0.5 point. So I wonder the initial parameters are the same ,just the difference of random seed. Why can it cause such different results?

**Comment by airsplay** [comment]
In general, GPU computation is non-deterministic to speed the training process up, even the seed is set. Sometimes, the same seed would give the same results, but it is just by luck. The results would change if you upgrade CUDA / upgrade PyTorch / or just run the same experiment at a different time.

Please read [this issue](https://github.com/airsplay/lxmert/issues/5) for detailed explanation.
